src/Universum/Functor/Reexport.hs

```diff
@@ -4,10 +4,12 @@ module Universum.Functor.Reexport
        ( module Control.Arrow
        , module Data.Bifunctor
        , module Data.Functor
+       , module Data.Functor.Compose
        , module Data.Functor.Identity
        ) where
 
 import Control.Arrow ((&&&))
 import Data.Bifunctor (Bifunctor (..))
 import Data.Functor (Functor (..), void, ($>), (<$>))
+import Data.Functor.Compose (Compose (..))
```

Comment:
Unfortunately, this will fail for GHC 7.10.3 because `Compose` appeared in base only since version 4.9.0.0. So you need to add CPP pragma for this reexport.

Reply:
But I tested it locally and it doesn't fail..

Reply:
Strange... Okay, let's wait for CI result then. If it works for CI, then we can keep it as it is.